interface/src/app/tribe_client/geneset-search-bar.tpl.html

```diff
@@ -1,12 +1,11 @@
 <div>
   <form class="form-inline">
     <label>Organism:</label>
-    <select ng-disabled="true">
-      <option ng-selected="true" value="organism">{{organism}}</option>  
-    </select>
+    <i>{{$ctrl.organism}}</i>&nbsp;
+    <input ng-model="$ctrl.search.query" placeholder="Search text">
```

Comment:
Search feedback looks good here now. I'm still not getting a reply when I search on "mexR" or "PA2196," but I can see now that it's trying. Perhaps this is an Elasticsearch configuration or performance issue? At any rate, this doesn't appear to be an issue with the code changed in this PR.